.changeset/calm-cougars-learn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/adapter-node': minor
```

Comment:
this change could be considered a feature or a fix. I'm not sure it's big enough for a new minor release, so a patch release is probably more appropriate
```suggestion
'@sveltejs/adapter-node': patch
```